recommender/translations/config.yaml

```diff
@@ -0,0 +1,95 @@
+# Configuration for i18n workflow.
```

Comment:
please remove this file unless absolutely needed

Reply:
it is needed by `i18n_tool`. I've cleaned it to the minimum